### PR TITLE
check test-infra bazel version and fail on not meeting minimum suppor…

### PR DESCRIPTION
**Pull request by BenTheElder**
…ted version

fixes https://github.com/kubernetes/test-infra/issues/5213#issuecomment-340110749

/area bazel

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -1,3 +1,15 @@
+http_archive(
+    name = "io_kubernetes_build",
+    sha256 = "8e49ac066fbaadd475bd63762caa90f81cd1880eba4cc25faa93355ef5fa2739",
+    strip_prefix = "repo-infra-e26fc85d14a1d3dc25569831acc06919673c545a",
+    urls = ["https://github.com/kubernetes/repo-infra/archive/e26fc85d14a1d3dc25569831acc06919673c545a.tar.gz"],
+)
+
+load("@io_kubernetes_build//defs:bazel_version.bzl", "check_version")
+# Ensure minimim supported bazel version
```
Comment (ixdy) [inline WORKSPACE]: minimum

File: WORKSPACE [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+# TODO(bentheelder): keep this up to date!
```
Comment (ixdy) [inline WORKSPACE]: this seems like an unsatisfiable TODO

File: WORKSPACE (hunk 1 above)
Comment (BenTheElder) [reply WORKSPACE]: oops, fixed.

File: WORKSPACE (hunk 2 above)
Comment (BenTheElder) [reply WORKSPACE]: should I remove it then? someone should be updating these when we upgrade deps, we can pretty easily check which is minimum with planter?

Comment (BenTheElder) [reply WORKSPACE]: removed.

**Comment by xiangpengzhao** [comment]
:100:

**Pull request merged**